build.gradle

```diff
@@ -373,6 +380,13 @@ dependencies {
     implementation group: 'org.apache.tomcat.embed', name: 'tomcat-embed-core', version: '9.0.86'
     implementation group: 'org.apache.tomcat.embed', name: 'tomcat-embed-websocket', version: '9.0.86'
 
+    //evidence-share depenedencies
+    implementation group: 'com.github.hmcts', name: 'cmc-pdf-service-client', version: '7.0.1'
+    implementation group: 'com.github.hmcts', name: 'send-letter-client', version: '3.0.23'
```

Comment:
might be a case for moving to EM document-generation with docmosis, rather than using the old pdf-service